cmd/otelcol/config/collector/agent_config.yaml

```
@@ -159,6 +159,8 @@ exporters:
   # Entities (applicable only if discovery mode is enabled)
   otlphttp/entities:
     logs_endpoint: "${SPLUNK_INGEST_URL}/v3/event"
+    # Use instead when sending to gateway
+    # endpoint: http://${SPLUNK_GATEWAY_URL}:4318
```

```suggestion
    # Use instead when sending to gateway
    # endpoint: ${SPLUNK_GATEWAY_URL}:4318
```
Is `http://` required here? Same question for other usages. I'm wondering why the value provided isn't consistent with usage of other environment variables.

---

yes, http is needed. By default OTLP receiver on GW is not configured for TLS.

---

We don't use the `SPLUNK_GATEWAY_URL` env var in the helm chart. Instead we dynamically configure the exporters to point to the gateway service. See https://github.com/signalfx/splunk-otel-collector-chart/blob/6d10b38a5d28e326d52f0a16dc22475a7f50a7b4/helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl#L852. User doesn't have to do anything other than setting `gateway.enabled` to `true`

---

@dmitryax 
Yes, makes sense.
PR is updated, please review